frappe/__init__.py

```diff
@@ -935,8 +934,6 @@ def clear_cache(user: str | None = None, doctype: str | None = None):
 	if hasattr(local, "website_settings"):
 		del local.website_settings
 
-	clear_routing_cache()
```

Comment:
We should retain this since all keys will only be deleted if user/doctype isn't passed

Reply:
Added it back, harmless.